README.md

```diff
@@ -239,5 +239,6 @@ Extensions for formatting flake8 output.
 - [flake8-gl-codeclimate](https://github.com/awelzel/flake8-gl-codeclimate) - Generates Gitlab Code Quality artifacts.
 - [flake8-html](https://github.com/lordmauve/flake8-html) - Generates an HTML report.
 - [flake8-json](https://github.com/PyCQA/flake8-json) - Generates JSON output.
+- [flake8-github](https://pypi.org/project/flake8-github) - Generates GitHub annotation for PR's.
```

Comment:
Please, add a link to github page of the project rather than PyPI, as for all other packages in the list.

Reply:
Ohh, you're right. Thanks!